### PR TITLE
Add docs for RQL endpoints

### PR DESCRIPTION
**Pull request by jondeandres**

### CODE REVIEW

File: api/rql.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# RQL
+
+These calls require a project-level access token with `read` scope, which should be provided in the query string. The prefix for all URLs is `https://api.rollbar.com`.
+
+## Create a RQL job
+
+Creates a new RQLjob if `force_refresh` is true or one for the same `query_string` does not already exist. Returns the found or newly-created RQL job object.
```
Comment (brianr) [inline api/rql.md]: missing a space: "Create a new RQL job"

**Comment by brianr** [comment]
Nice, I think this is good to go. We can tweak after release.

File: api/rql.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+Returns the job metadata and, optionally, the current value of the job result.
+
+    GET /api/1/rql/job/:job_id
+
+### Query string parameters
+
+Name | Type | Description
+-----|------|-------------
+`expand`|`string`|If the value is 'result', the response will contain the job result
```
Comment (brianr) [inline api/rql.md]: Didn't we remove this param?

File: api/rql.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+  "err" 0,
+  "result": {
+    "id": 123,  // job id
+    "project_id": 456,
+    "query_string": "show tables",
+    "status": "new",
+    "job_hash": "abcdefabcdefabcdef",
+    "date_created": 1446598885,
+    "date_modified": 1446598885,
+    "result": {...} // A RQL job resource if expand=result is used in query string
```
Comment (brianr) [inline api/rql.md]: same comments comment as above about removing this

File: api/rql.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    "result": {
+      "rows": [{...}],
+      "selectionColumns": [...],
+      "columns": [...],
+      "errors": [],
+      "warnings": [],
+      "rowcount": 1,
+      "executionTime": 123
+    },
+    "job": {...} // A RQL job resource if result=job is set in the query string
```
Comment (brianr) [inline api/rql.md]: should say `expand=job`. Also, did we actually implement this?

**Comment by jondeandres** [comment]
Typos fixed, merging.

**Pull request merged**